CHANGELOG.md

```diff
@@ -6,8 +6,15 @@
 
 ### Feature
 
+- Add enable/disable toggle for sorting section in the search block, and fix missing sorting for the top variation @kreafox
```

Comment:
@tiberiuichim @kreafox can we have a single changelog entry here please that says "Add search block"?